app/controllers/ops_controller/settings/common.rb

```diff
@@ -179,6 +179,14 @@ def pglogical_subscriptions_form_fields
     replication_type = MiqRegion.replication_type
     subscriptions = replication_type == :global ? PglogicalSubscription.all : []
     subscriptions = get_subscriptions_array(subscriptions) unless subscriptions.empty?
+    if replication_type == :global
+      subscriptions.each do |h|
+        region = MiqRegion.where(:region => h['provider_region']).first
```

Comment:
No client side-side encryption - rather secure channel to server expected - same as the other forms.